frontend/dockerfile/cmd/dockerfile-frontend/hack/release

```diff
@@ -75,6 +75,18 @@ if [[ "$RELEASE" = "true" ]] && [[ "$GITHUB_ACTIONS" = "true" ]]; then
   nocacheFilterFlag="--no-cache-filter=base"
 fi
 
+outputFlag="type=image,$pushFlag"
+if [ "$PUSH" = "push" ] && [ "$GITHUB_ACTIONS" = "true" ]; then
+  outputFlag="${outputFlag},\"annotation.org.opencontainers.image.title=Dockerfile Frontend\""
+  if [ -n "$GITHUB_SHA" ]; then
+    outputFlag="${outputFlag},annotation.org.opencontainers.image.revision=$GITHUB_SHA"
+  fi
+  if [ -n "$GITHUB_REPOSITORY" ] && [ -n "$GITHUB_SERVER_URL" ]; then
+    outputFlag="${outputFlag},annotation.org.opencontainers.image.source=$GITHUB_SERVER_URL/$GITHUB_REPOSITORY"
+    outputFlag="${outputFlag},annotation.org.opencontainers.image.url=$GITHUB_SERVER_URL/$GITHUB_REPOSITORY"
+  fi
+fi
```

Comment:
I think we could mutualize with what has been done in https://github.com/moby/buildkit/pull/3554.

Maybe have something common in `hack/util`?

Reply:
I don't think there's much in common; moving the small amount of commonality to a library would increase complexity.

The only annotations that frontend and buidkit have in common are `org.opencontainers.image.source=$GITHUB_SERVER_URL/$GITHUB_REPOSITORY`, `org.opencontainers.image.url=$GITHUB_SERVER_URL/$GITHUB_REPOSITORY`, and `org.opencontainers.image.revision=$GITHUB_SHA`.  The other annotations, `org.opencontainers.image.version` and `org.opencontainers.image.title`, have different values.

We would either need to make a function that accepts inputs for the title and version, or not move title and version to the library and only have library function handle source, url, and revision. I think either approach results in a net increase in complexity.

I'll implement the library approach if that's what it takes to get this improvement in, but I don't think doing so would be a win.

Reply:
We can look at this as follow-up if you want

Reply:
Sounds good to me